packages/skott/bin/cli.ts

```diff
@@ -28,113 +28,138 @@ function readManifestVersion(): string {
+ * @throws {Error} - Throws an error if the given value is not a number.
+ */
+function commanderParseInt(value: string): number {
+  // parseInt takes a string and a radix
+  const parsedValue = parseInt(value, 10);
+  if (isNaN(parsedValue)) {
+    throw new Error("Not a number.");
+  }
 
-  .describe("Start the Skott analysis to fully build the graph")
+  return parsedValue;
+}
 
+/**
+ * Function similar to kotlin's trimMargin
+ * This is really helpful to print multiline string while keeping code indentation readable
+ * Example:
+ * trimMargin('|')(`
+ *     |this is a super long multiline string
+ *     |defined on multiple lines
+ *     |while well formatted in the code
+ * `)
+ * will return:
+ * this is a super long multiline string
+ * defined on multiple lines
+ * while well formatted in the code
+ * @param delimiter
+ */
+export function trimMargin(delimiter = "|"): (str: string) => string {
+  const reg = new RegExp(`^\\s*\\${delimiter}`, "g");
+
+  return (str: string): string =>
+    str
+      .split("\n")
+      .map((line) => line.replace(reg, ""))
+      .join("\n");
+}
+
+const cli = new Command();
+
+cli
+  .version(readManifestVersion())
+  .description("Start the Skott analysis to fully build the graph")
   .option(
     "-b, --includeBaseDir",
     "Include the base directory name for each graph node. Relative to the entrypoint base directory or the current working directory.",
     false
   )
-
   .option(
-    "-c, --exitCodeOnCircularDependencies",
+    "-c, --exitCodeOnCircularDependencies <code>",
     "Specify the exit code to use when circular dependencies are found",
+    commanderParseInt,
     1
   )
-
   .option(
-    "-d, --circularMaxDepth",
+    "-d, --circularMaxDepth <maxDepth>",
     "Define the max depth of the nested circular dependencies search",
+    commanderParseInt,
     Number.POSITIVE_INFINITY
   )
-
   .option(
-    "-e, --fileExtensions",
-    "File extensions to explore when building the graph",
+    "-e, --fileExtensions <extensions>",
+    'File extensions to explore when building the graph, separated by a comma (ex: ".js,.ts)"',
     [...kExpectedModuleExtensions].join(",")
   )
-
   .option(
-    "-m, --displayMode",
-    "Either display the result of the analysis as a 'graph', as a 'file-tree' or 'raw'",
+    "-m, --displayMode <mode>",
+    "Either display the result of the analysis as a graph, as a file-tree or raw",
     "webapp"
   )
-
   .option(
     "-n, --trackBuiltinDependencies",
     "Enable Node.js builtin dependency tracking",
     false
   )
-
   .option(
     "-i, --incremental",
     "(Experimental) Enable incremental mode. Only the files that changed since the last run will be analyzed.",
     false
   )
-
   .option(
     "-it, --trackTypeOnlyDependencies",
-    "Enable dependency tracking for Typescript 'import type' statements",
-    true
+    "Enable dependency tracking for Typescript import type statements",
+    false
```

Comment:
@ACHP just noticed that this changed through the refactoring (from "true" to "false"). Could you please put it back to true so that it stays in sync and consistent with the [API default config](https://github.com/antoine-coulon/skott/blob/7d337fa552dc24a8f00a1fc5011b82fac2f7be40/packages/skott/src/skott.ts#L99)?

Reply:
solved in https://github.com/antoine-coulon/skott/pull/88/commits/a1054c821d27209fdd2b37fcefff0c5133eeefe2